### PR TITLE
STAC-5314 Publish java trace client to S3

### PR DESCRIPTION
**Pull request by jglad20**
- Changed Gradle Wrapper to use the latest 5.5.1

- Removed classifier= unbundled for jar build

- Changed publish target from artifactory/bintray to S3  

- Applied branding

- for publish of agent jar to S3 use ./gradlew :dd-java-agent:publish

### CODE REVIEW

File: dd-trace-java.gradle [hunk 1]
```diff
@@ -54,7 +54,7 @@ buildScan {
 
 wrapper {
   distributionType = Wrapper.DistributionType.ALL
-  version = '5.4.1'
+  version = '5.5.1'
```
Comment (Voronenko) [inline dd-trace-java.gradle]: Why 5.5.1 ?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
```
Comment (Voronenko) [inline gradle/wrapper/gradle-wrapper.properties]: What was added in 5.5.1 over 5.4.1 ?

File: gradle/publish.gradle [hunk 3]
```diff
@@ -1,34 +1,8 @@
-/*
```
Comment (Voronenko) [inline gradle/publish.gradle]: We are extending opensourced client, thus there will be upstream changes from time to time.
As long as it does not affect result - I would leave as much of lines as possible intact.

And perhaps extend S3 way of publishing as a separate gradle publishing task.
Who knows, maybe version will be published to artifactory as well.

File: gradlew [hunk 4]
```diff
@@ -7,7 +7,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
```
Comment (Voronenko) [inline gradlew]: This change is not needed.

File: gradlew [hunk 5]
```diff
@@ -185,4 +185,4 @@ if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
   cd "$(dirname "$0")"
 fi
 
-exec "$JAVACMD" "$@"
+exec "$JAVACMD" "$@"
```
Comment (Voronenko) [inline gradlew]: This change is not needed.

**Review by Voronenko — CHANGES_REQUESTED** [review]

File: dd-trace-java.gradle (hunk 1 above)
Comment (jglad20) [reply dd-trace-java.gradle]: rolled back

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (jglad20) [reply gradle/wrapper/gradle-wrapper.properties]: rolled back

File: gradle/publish.gradle (hunk 3 above)
Comment (jglad20) [reply gradle/publish.gradle]: Added Artifactory/Bintray along with S3 Maven repo

File: gradlew (hunk 4 above)
Comment (jglad20) [reply gradlew]: rolled back

File: gradlew (hunk 5 above)
Comment (jglad20) [reply gradlew]: rolled back

**Review by Voronenko — APPROVED** [review]

**Pull request merged**